### DISCUSSION TITLE
Respect for vico! Q:how to disable vico animation for linechart

### DISCUSSION THREAD

**Discussion by zch1234qq** [patrykandpatrick/vico#341] [Questions] [answered]
I want to plot  hundreds of chart more Efficiently，thanks bro!

**Comment by patrickmichalik** [comment]
Hello! Are you referring to the animation that occurs on chart creation? If so, you can turn it off by setting `runInitialAnimation`—which is a parameter of the `Chart` composable function and a field of `BaseChartView`—to `false`.

**Reply by zch1234qq** [reply]
I mean the animation after chart model updated by chartModelProducer. (when chart runing)

**Reply by Aatif-Thinkitive** [reply]
yes i think you can disable it by setting `runInitialAnimation` to `false`

**Reply by patrickmichalik** [reply, marked as answer]
Got it, @zch1234qq. The initial animation is a special kind of difference animation. Because of this, disabling standard difference animations will also mean disabling the initial animation. To do so, you can set the `diffAnimationSpec` parameter of the `Chart` composable function to `snap()` or call `BaseChartView#setDiffAnimationDuration` with zero as its argument. At this point, though, your `ChartEntryModelProducer` won’t be doing much, so you may prefer to create `ChartEntryModel` instances yourself. If you’re using Jetpack Compose, you can use state to ensure that your chart is always displaying the latest data.
